src/sentry/api/event_search.py

```diff
@@ -240,7 +242,13 @@ def is_negation(self):
 class SearchKey(namedtuple("SearchKey", "name")):
     @cached_property
     def is_tag(self):
-        return TAG_KEY_RE.match(self.name) or self.name not in SEARCH_MAP
+        return TAG_KEY_RE.match(self.name) or (
```

Comment:
@Zylphrex I think we can remove this since measurements are no longer in tags.

Reply:
That's not what this line does. When resolving column names, the resolution order is now

1. check if it is a known column
2. check if it is a measurement (new)
3. check if it is a tag of the form `tag[...]`
4. assume it is a tag

This here impacts https://github.com/getsentry/sentry/blob/master/src/sentry/api/event_search.py#L810-L852 as tags get different treatment from columns when generating the conditions for snuba as the default value for tags is the empty string (e.g. `""`) whereas measurements behave like columns in this sense that the default value is `NULL`. So this is necessary to ensure that measurements are correctly handled when generating the conditions.